test/test_helper.rb

```diff
@@ -30,6 +30,8 @@
   c.stubbing_non_public_method = :prevent
 end
 
+Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
```

Comment:
We should probably set this in the exercism config gem.

Reply:
Does this get set in dev or production anywhere currently?

Reply:
It defaults to `Rails.cache`. I don't yet set this anywhere.